manifests/m/Microsoft/PIX/24.3.8.1/Microsoft.PIX.locale.en-US.yaml

```
@@ -0,0 +1,18 @@
+# Created with komac v2.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Microsoft.PIX
+PackageVersion: 24.3.8.1
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+Author: Microsoft
+PackageName: Microsoft PIX 2403.08
```

```suggestion
PackageName: Microsoft PIX
```
Including the version here will cause other versions to not be mapped correctly